web/templates/message_body.hbs

```diff
@@ -1,7 +1,7 @@
 {{#if include_sender}}
     {{> message_avatar ~}}
 {{/if}}
-<span class="message_sender no-select">
+<span class="message_sender">
     {{#if include_sender}}
         <span class="sender_info_hover sender_name auto-select" role="button" tabindex="0">
```

Comment:
So I'm pretty sure the internal design here is that we had `no-select` on the outer element and `auto-select` on these inner elements to override it.

And I think the bug here is that in some browsers, this doesn't actually work; the override gets ignored.

We can presumably just push down the `no-select` to whatever elements inside we want to not be selectable; I don't remember the intent but it looks like the edited notice and interstitial whitespace are the only possible candidates; my vague recollection of history is this had to do with whitespace control and probably was fixed by the grid changes.

So I think the solution is to just make sure we actually remove the `auto-select` bits as well?

Reply:
Yeah, the edit notices have auto-select too.
```
web/templates/edited_notice.hbs:<div class="message_edit_notice auto-select" data-tippy-content="{{t 'Last edited {last_edit_timestr}' }}">
web/templates/edited_notice.hbs:<div class="message_edit_notice auto-select" data-tippy-content="{{t 'Last moved {last_edit_timestr}' }}">
web/templates/edited_notice.hbs:<div class="message_edit_notice auto-select" data-tippy-content="{{t 'Last edited {last_edit_timestr}' }}">
```